src/components/DialogToolbarContent.vue

```diff
@@ -7,10 +7,11 @@
         </div>
       </div>
     </el-row>
-    <el-row class="icon-group" v-if="showIcons">
-      <el-button class="icon-transform" icon="el-icon-copy-document" size="medium" type="text" @click="maximise"></el-button>
-      <el-button icon="el-icon-remove-outline" size="medium" type="text" @click="minimise"></el-button>
-      <el-button icon="el-icon-close" size="medium" type="text" @click="close"></el-button>
+    <el-row class="icon-group" >
+      <el-button icon="el-icon-full-screen" size="medium" type="text" v-if="showFullscreenIcon" @click="onFullscreen"></el-button>
```

Comment:
One small thing, should full screen button stay in the top right?

Reply:
Actually don't worry I'll just accept this and create a new PR with fullscreen button always on the right

Reply:
I think the close button should be on the right but I am open to change it if that is agreed upon.

Reply:
Yeah it's not a biggie, can always change it if design finds it is better on the right.

Reply:
Happy to accept merge this in if that's what you are waiting for? But I don't have write access.